installer/openshift/templates/configmap.yml.j2

```diff
@@ -17,6 +17,11 @@ data:
     CLUSTER_HOST_ID = socket.gethostname()
     SYSTEM_UUID = '00000000-0000-0000-0000-000000000000'
     
+    # Token-based authentication performs a hash check on HTTP headers expected to be consistent, including the remote host IP.
```

Comment:
I totally agree with adding this as a default remote host header but I don't think that it needs such a thorough explanation.

I actually think it might be better served to have this change made in our defaults: https://github.com/ansible/awx/blob/devel/awx/settings/defaults.py#L151